components/providers/AccordionLayout.vue

```diff
@@ -1,72 +1,113 @@
+      <code class="accordion-layout__code-cell">
+        <pre class="accordion-layout__code-cell__line">{{ installation }}</pre>
+      </code>
+      <cv-button
+        title="Copy"
+        class="accordion-layout__copy-button"
+        @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} install code`, 'providers')]"
+      >
+        Copy
+      </cv-button>
+    </div>
+    <div class="accordion-layout__code-block">
+      <code class="accordion-layout__code-cell">
+        <template v-for="(line, index) in helloWorldExample">
+          <!-- eslint-disable vue/no-v-html -->
+          <pre v-if="line.startsWith('#')" :key="line" class="accordion-layout__code-cell__comment" v-html="line" />
+          <pre v-else :key="index" class="accordion-layout__code-cell__line" v-html="line" />
+          <!-- eslint-enable -->
+        </template>
+      </code>
+      <cv-button
+        title="Copy"
+        class="accordion-layout__copy-button"
+        @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} helloWorldExample`, 'providers')]"
+      >
+        {{ copyCodeCta }}
+      </cv-button>
+    </div>
+    <div class="accordion-layout__cta-group">
+      <AppCta
+        v-for="cta in validCtas"
+        :key="cta.label"
+        kind="ghost"
+        v-bind="cta"
+      />
+    </div>
   </article>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
 import { Component, Prop } from 'vue-property-decorator'
+import { NavLink } from '~/constants/menuLinks'
 
-// TODO: It is not possible to extract the interface of the component props
-// so we need this redundant interface to do it. Explore if it is worth
-// removing the @Prop decorator in favour of:
-// https://class-component.vuejs.org/guide/props-definition.html
 interface AccordionLayoutProps {
-  image: string,
-  description: string,
-  cta: {
-    url: string,
-    label: string
+  title: string
+  image: string | null
+  description: string
+  installation: string
+  websiteCta: {
+    label: string | null
+    url: string | null
   }
+  docsCta: {
+    label: string | null
+    url: string | null
+  }
+  sourceCta: {
+    label: string | null
+    url: string | null
+  }
+  helloWorldExample: string[]
 }
 
 export { AccordionLayoutProps }
 
 @Component
 export default class AccordionLayout extends Vue implements AccordionLayoutProps {
-  @Prop(String) image!: string
+  @Prop(String) title!: string
   @Prop(String) description!: string
-  @Prop(Object) cta!: {
-    url: string,
-    label: string
+  @Prop(String) image!: string
+  @Prop(String) installation!: string
+  @Prop(Object) websiteCta!: NavLink
+  @Prop(Object) docsCta!: NavLink
+  @Prop(Object) sourceCta!: NavLink
+  @Prop(Array) helloWorldExample!: string[]
+
+  get validCtas () {
+    return [this.websiteCta, this.docsCta, this.sourceCta].filter(cta => cta.url !== null)
+  }
+
+  copyCodeCta = 'Copy'
+
+  copyToClipboard (e: { target: any }): void {
```

Comment:
`void` return type is inferred, you don't need to add it

```suggestion
  copyToClipboard (e: { target: any }) {
```

Reply:
Fixed in https://github.com/Qiskit/qiskit.org/pull/3000/commits/88ac621a65a1abc7d1e8637babbe6b846672e819